tests/languages/docker_test.py

```diff
@@ -21,3 +201,19 @@ def invalid_attribute():
         getgid=invalid_attribute,
     ):
         assert docker.get_docker_user() == ()
+
+
+def test_docker_is_not_rootless():
+    with mock.patch.object(docker, 'cmd_output', return_value=(0, DOCKER_NOT_ROOTLESS_SYSTEM_INFO, '')):
+        assert docker.docker_is_rootless() == False
```

Comment:
The idiomatic way is to do
```suggestion
        assert not docker.docker_is_rootless()
```